bigtable-hbase-2.x-parent/bigtable-hbase-2.x-integration-tests/pom.xml

```diff
@@ -160,9 +160,9 @@ limitations under the License.
                   </includes>
                   <excludedGroups>KnownEmulatorGap,KnownGap</excludedGroups>
                   <systemPropertyVariables>
-                    <google.bigtable.test_env>bigtable</google.bigtable.test_env>
                     <google.bigtable.project.id>fake-project</google.bigtable.project.id>
                     <google.bigtable.instance.id>fake-instance</google.bigtable.instance.id>
+                    <google.bigtable.test_env>bigtable</google.bigtable.test_env>
```

Comment:
nit - can revert this?

Reply:
This was updated in https://github.com/googleapis/java-bigtable-hbase/pull/3452/files to use bigtable/minicluster instead of connection.impl.. I think I might have missed this pom file